beard.js

```diff
@@ -109,8 +112,8 @@ module.exports = function(cache = {}, opts = { cache: true }) {
 
   function compiled(path, parentPath) {
     const fullPath = resolvePath(path, parentPath);
-    const str = opts.cache
-      ? cache[fullPath]
+    const str = opts.cache === true
```

Comment:
@shanebo Does this need to be opts.cache === true and not just opts.cache?

Reply:
@joeosburn good catch. I did that while troubleshooting why the cache wasn't working. But I realized that setting default arg values don't merge with what passes in, which is a bummer.